content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md

```diff
@@ -80,6 +88,24 @@ Maximum concurrency was measured using multiple repositories, job duration of ap
 
 {%- endif %}
 
+
+{%- ifversion ghes = 3.6 %}
+
+{% data reusables.actions.hardware-requirements-3.6 %}
+
+{% data variables.product.company_short %} measured maximum connected runners using multiple repositories, job duration of approximately 10 minutes, and 10 MB artifact uploads. You may experience different performance depending on the overall levels of activity on your instance.
+
+{% note %}
+
+**Note:** Beginning with {% data variables.product.prodname_ghe_server %} 3.6, {% data variables.product.company_short %} documents connected runners as opposed to concurrent runners. Connected runners resprents the most runners you can connect and expect to utilize.
```

Comment:
```suggestion
**Note:** Beginning with {% data variables.product.prodname_ghe_server %} 3.6, {% data variables.product.company_short %} documents connected runners as opposed to concurrent jobs. Connected runners represents the most runners you can connect and expect to utilize.
```

We also might want to add a sentence like "Connecting more runners than you expect to utilize can negatively impact performance."